x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -505,6 +511,40 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
             rowHeightsOptions={props.rowHeightsOptions}
             onColumnResize={onColumnResize}
             ref={dataGridRef}
+            renderCustomGridBody={
+              props.dynamicRowHeight
+                ? ({ visibleColumns: _visibleColumns, Cell }) => {
+                    return (
+                      <>
+                        {oldAlertsData
+                          .concat(
+                            isLoading
+                              ? Array.from({ length: pagination.pageSize - oldAlertsData.length })
+                              : []
+                          )
+                          .map((_row, rowIndex) => (
+                            <Row
+                              role="row"
+                              className={`euiDataGridRow ${
+                                rowIndex % 2 !== 0 ? 'euiDataGridRow--striped' : ''
+                              } ${actualGridStyle.rowClasses?.[rowIndex] ?? ''}`}
+                              key={rowIndex}
+                            >
+                              {_visibleColumns.map((_col, colIndex) => (
+                                <Cell
+                                  colIndex={colIndex}
+                                  visibleRowIndex={rowIndex}
+                                  key={`${rowIndex},${colIndex}`}
+                                  style={{ flexShrink: 0 }}
```

Comment:
@umbopepato Is this the collapse issue you mentioned in Slack? I'm not totally sure why this extra style would be necessary 🤔  I'm seeing our loading skeleton components as working in the default EUI example:

https://codesandbox.io/s/gracious-cdn-2q8yf9?file=/demo.tsx

Is there some custom CSS that your plugin is setting on the datagrid?

Reply:
You're right, it's not necessary (I introduced it when I was experiencing an issue with the last column overgrowing and squeezing the other ones past their intended widths, which actually stopped happening after a couple of upstream pulls). It turns out that in our case the loaders collapse because when using `defaultHeight: 'auto'`, `.euiDataGridRowCell__content` elements don't get these styles like in your codesandbox:

<img width="330" alt="image" src="https://github.com/elastic/kibana/assets/18363145/6c113ac1-b960-4e27-a48a-34fd2535d392">